cmd/configure/configure_test.go

```diff
@@ -106,7 +106,46 @@ func TestConfigFileFromEnvNoInteractive(t *testing.T) {
 	assertKeyValueInSection(t, defaultSection, "token", "token")
 }
 
-func TestCustomProfileConfigureNoInteractive(t *testing.T) {
+func TestEnvVarsConfigureNoInteractive(t *testing.T) {
+	ctx := context.Background()
+	tempHomeDir := setup(t)
+	cfgPath := filepath.Join(tempHomeDir, ".databrickscfg")
+	inp := getTempFileWithContent(t, tempHomeDir, "token\n")
+	defer inp.Close()
+	oldStdin := os.Stdin
+	t.Cleanup(func() { os.Stdin = oldStdin })
+	os.Stdin = inp
+
+	t.Setenv("DATABRICKS_HOST", "https://host")
+	t.Setenv("DATABRICKS_AUTH_TYPE", "metadata-service")
+	t.Setenv("DATABRICKS_METADATA_SERVICE_URL", "https://metadata")
```

Comment:
Please also set the `DATABRICKS_TOKEN` env var to some value to make sure it doesn't have precedence over the value that the user provides through stdin.

Reply:
With the current logic it will pick up the token from the env var and won't even try to ask for a new one from a user (or read from sdtin). From git blame I figured it is a feature, as it was added explicitly - the CLI is able to bootstrap the config based on the environment variables, and if there's enough of them, it will pretty much create a config automatically without any additional prompts. 

To me it feels a bit counter intuitive, I would expect `databricks configure` to always ask all of its inputs. With the current logic it should be named `databrikcs write-to-config` instead, not `configure` (since in some cases you don't even configure anything).

Would it make sense to remove reading from the environment completely?
https://github.com/databricks/cli/blob/3284a8c56c65cfa53cd098874f99b2b76714353e/cmd/configure/configure.go#L140

Reply:
And a side note: databricks vscode extension doesn't set the token env variable, only host with metadata

Reply:
There will be users who set these env vars and expect it to work (predominantly for automation), so let's keep it in tact. I have seen cases where the token is passed through stdin, the rest being set through env vars. Looking at the test context I figured the other env vars would also be masked out, but since it makes sense, disregard my comment.

Do we already have a test confirming that this behavior works? I.e. pass the host and token as env vars and the command silently succeeds and writes out a profile?

Reply:
Added a test to confirm that `configure` works with both token and host taken from the env vars